.evergreen/env.sh

```diff
@@ -19,5 +19,7 @@ if [[ "$OS" == "Windows_NT" ]]; then
     export PATH
     echo "updated path on windows PATH=$PATH"
 else
+    # Turn off tracing for the very-spammy nvm script.
```

Comment:
This is unrelated, but the spam was making finding the actual test failures annoying.